### PR TITLE
LPS-153775 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-153775

@david-gutierrez-mesa 
@Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 1]
```diff
@@ -1020,6 +1020,40 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-148426. The page creator can successfully publish a content page with commerce widgets."
+	@priority = "4"
+	test PublishPageWithCommerceWidgets {
+		task ("Given a page creator has a published content page with commerce widgets") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1026]: Given a page creator has a content page.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 2]
```diff
@@ -1020,6 +1020,40 @@ definition {
...
+			for (var widgetName : list "Open Carts,Placed Orders,Wish List Content,Wish Lists") {
+				PageEditor.addWidget(
+					portletName = "${widgetName}",
+					targetFragmentName = "Checkout");
+			}
+
+			PageEditor.publish();
+		}
+
+		task ("When the page creator accesses content page at view mode") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1042]: When the page creator adds commerce widgets to page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 3]
```diff
@@ -1020,6 +1020,40 @@ definition {
...
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+		}
+
+		task ("Then the page creator should see the commerce widgets") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1048]: The the page creator could publish the content page

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! PR updated. Could you please have a look at? Thanks!

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**